frontend/javascripts/oxalis/view/version_entry.js

```diff
@@ -96,6 +157,7 @@ function getDescriptionForBatch(actions: Array<ServerUpdateAction>): Description
 
   // If more than one createNode update actions are part of one batch, that is not a tree merge or split
   // an NML was uploaded.
+  // TODO: This could've also been an undo/redo action.
```

Comment:
Hmm, tricky. A simple solution would be to generalize the description message (e.g., "X nodes were added.") or to just leave this as is. Figuring out what the correct reason is would obviously be better, but I can't think of a good solution right now :/

Reply:
Yes, I think we cannot do much about it easily apart from generalizing the message as you suggested. I'll create an issue for that.

Reply:
For reference https://github.com/scalableminds/webknossos/issues/4883